data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1520,56 +1578,97 @@ descriptions:new_tribe {
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Toolsmithy
+            lore = pgettext("atlanteans_building", "People's power working for the benefit of all"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Toolsmithy
+            lore_author = pgettext("atlanteans_building", "Faded sign found on top of this building")
          }
       },
       {
          name = "atlanteans_weaponsmithy",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Weapon Smithy
-            purpose = pgettext("atlanteans_building", "Forges tridents to equip the soldiers and to train their attack in the dungeon.")
+            purpose = pgettext("atlanteans_building", "Forges tridents to equip the soldiers and to train their attack in the dungeon."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site:  Weapon Smithy
+            lore = pgettext("atlanteans_building", "You hear that sound of metal stridend? The weapon smith just forged a trident!"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Weapon Smithy
+            lore_author = pgettext("atlanteans_building", "Atlantean nursery rhyme")
+
          }
       },
       {
          name = "atlanteans_armorsmithy",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Armor Smithy
-            purpose = pgettext("atlanteans_building", "Forges shields that are used for training soldiers’ defense in the labyrinth.")
+            purpose = pgettext("atlanteans_building", "Forges shields that are used for training soldiers’ defense in the labyrinth."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Armor Smithy
+            lore = pgettext("atlanteans_building", "She twists and tweaks the metal into form, a soldier will it use as arm"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Armor Smithy
+            lore_author = pgettext("atlanteans_building", "Atlantean nursery rhyme")
          }
       },
       {
          name = "atlanteans_barracks",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Barracks
-            purpose = pgettext("atlanteans_building", "Equips recruits and trains them as soldiers.")
+            purpose = pgettext("atlanteans_building", "Equips recruits and trains them as soldiers."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Barracks
+            lore = pgettext("atlanteans_building", "See these fine silk tabards and those sharp tridents? This is you way to defend your tribe!"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Barracks
+            lore_author = pgettext("atlanteans_building", "Recruiting Poster")
```

Comment:
```suggestion
            lore = pgettext("atlanteans_building", "See these fine silk tabards and those sharp tridents? This is your way to defend your tribe!"),
            -- TRANSLATORS: Lore author helptext for an atlantean production site: Barracks
            lore_author = pgettext("atlanteans_building", "Recruiting poster")
```

Reply:
"These are your way…"?

Reply:
@tothxa  any better idea from some old / current recruiting poster?

Reply:
It's fine. I'm just asking whether it's more appropriate to use plural for the second sentence too. I guess it makes sense with singular too, but plural seems more natural to me.